### ISSUE TITLE
ANT?

### ISSUE THREAD

**Issue by Prutsium** [A-damW/JBD_BMS_BLE_VESC_EXPRESS_BRIDGE#4]
Any plans on ANT BMS?

**Comment by A-damW** [comment]
> Any plans on ANT BMS?

Yes, the main thing stopping development for ANT is, I don't own any ANT bms, the cheapest ANT I have found is $40-50USD,

I started with JBD because I can buy them all day for $10-20USD (14s-17s)

My entire philosophy for this project is thus: 
1. Your motor controller(VESC©, of course!) is able to control ALL of the power flowing from, and to the battery(Acceleration, Regeneration, Just like a bms, huh.)

2. VESC© software/firmware is able to limit power(in and out of the battery), and protect the battery pack based on individual cell voltage limits(Just like a bms, huh.)

3. Understanding that the motor controller does NOT need some other device(BMS), to CONTROL the power, just knowledge/data of the individual cell voltages means we can use a cheap BMS, and forward the cell data to the controller so the controller is aware of the actual battery state, and act accordingly.

Short rant, summary: why pay as much, or more for a bms than a controller, when the controller(VESC©, of course!) can do everything a bms can do, if it the controller has battery cell data?!?!?
